pkg/controller/master/backup/backup.go

```diff
@@ -804,7 +804,7 @@ func (h *Handler) deleteVMBackupMetadata(vmBackup *harvesterv1.VirtualMachineBac
 		return err
 	}
 
-	destURL := filepath.Join(metadataFolderPath, getVMBackupMetadataFileName(vmBackup.Namespace, vmBackup.Name))
+	destURL := filepath.Join(metadataFolderPath, vmBackup.Namespace, vmBackup.Name)
```

Comment:
deleteVMBackupMetadata seems to generate a destURL that is asymmetrical to uploadVMBackupMetadata.

deleteVMBackupMetadata constructs `.../metadataFolderPath/vmBackup.Namespace/vmBackup.Name` but uploadVMBackupMetadata calls getVMBackupMetadataFilePath which returns `.../metadataFolderPath/vmBackup.Namespace/vmBackup.Name.cfg` (with .cfg extension).

Is that intentional?

Reply:
Yeah, I forgot to change this one. Updated it. Thank you.